DeAceroDLabs/marketplace-frontend
# Feature/dmp 493 and Feature/dmp 494 Logout

## What is done

[[DMP-493] Logout Tooltip](https://deacero.atlassian.net/browse/DMP-493)
[[DMP-494] Logout with auth0](https://deacero.atlassian.net/browse/DMP-494)

## How is done

- Added tooltip that logs out, and navigates to /conocenos and /ayuda to `Header` component.
- Added `closeWithX` prop to  `Tooltip` component.

## How to test

1. Run the project via `npm start`
2. login.
3. You should see the Home Screen. 
4. Click on `Hola, your_name`. You should see the tooltip.
5. Click on Ayuda to navigate to /ayuda.
6. Click on Conocenos to navigate to /conocenos.
7. Click on Salir to logout.
